StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/RotatingCardBrandsView.swift

```diff
@@ -50,6 +50,8 @@ class RotatingCardBrandsView: UIView {
                     return 5
                 case .unionPay:
                     return 6
+                case .cartesBancaires:
+                    return nil
```

Comment:
Why nil?

Reply:
We don't want CB to show up in the RotatingCardBrandsView, matching our behavior on the web. I'll add a comment though, I had to dig a bit to figure this out.